tests/test_outline.py

```diff
@@ -0,0 +1,82 @@
+from unittest.mock import Mock, patch
+
+import pytest
+from pydantic import BaseModel
+
+from outlines import Outline
+
+
+class OutputModel(BaseModel):
+    result: int
+
+
+def template(a: int) -> str:
+    return f"What is 2 times {a}?"
+
+
+def test_outline():
+    mock_model = Mock()
+    mock_generator = Mock()
+    mock_generator.return_value = '{"result": 6}'
+
+    with patch(
+        "outlines.generate.json", return_value=lambda *args, **kwargs: mock_generator()
+    ):
+        outline_instance = Outline(mock_model, template, OutputModel)
+        result = outline_instance(3)
```

Comment:
And add an assertion to make sure it's an instance of a Pydantic `BaseModel` once you make the change requested above.